core/bin/external_node/src/main.rs

```diff
@@ -177,12 +177,7 @@ async fn init_tasks(
     let tree_handle =
         task::spawn(metadata_calculator.run(tree_pool, prover_tree_pool, tree_stop_receiver));
 
-    let consistency_checker_handle = if !config.optional.experimental_multivm_support {
```

Comment:
Is this variable used anywhere now? If not, probably it makes sense to remove it altogether (I think it's undocumented, so no changes in docs/announcements required).

Reply:
No, will remove